fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1220,6 +1221,9 @@ def _lazy_init(self) -> None:
             self._set_is_root()
             self._setup_streams()
             self._setup_output_hook_list()
+            for module_name, module in self.named_modules():
+                if isinstance(module, FullyShardedDataParallel):
+                    module._module_fqn = module_name
```

Comment:
These only needed for debugging (when _FSDP_DEBUG is set in P841842878)

Reply:
Deleted.